src/redux-state/selectors/island.ts

```diff
@@ -5,7 +5,6 @@ import { IslandModeType } from "redux-state/slices/island"
 export const selectIslandMode = createIslandSelector("mode")
 export const selectIslandOverlay = createIslandSelector("overlay")
 export const selectIslandZoomLevel = createIslandSelector("zoomLevel")
-export const selectRealmPanelVisible = createIslandSelector("realmPanelVisible")
```

Comment:
This selector is necessary in this situation:

[screen-capture (16).webm](https://github.com/tahowallet/dapp/assets/73061939/4cbefd88-d407-4d27-84e2-05b297744e57)

When clicks on the "Close view" button, whole top part of left realm panel disappears and we can't just check if `displayedRealmId !== null`. The modal should stay the same height and have data visible until it closes.

Also it looks like there is a delay after click on close button